multichain-testing/scripts/ymax-tool.ts

```diff
@@ -118,8 +138,14 @@ const main = async (
     connectWithSigner = SigningStargateClient.connectWithSigner,
   } = {},
 ) => {
-  const [volume] = argv.slice(2);
-  if (!volume) throw Error(`USAGE: ${argv[1]} 123.45`);
+  // Extract volume and check for the --skip-poll flag
+  const args = argv.slice(2);
+  const skipPoll = args.includes('--skip-poll');
+  // Remove the flag from the args if present
+  const filteredArgs = args.filter(arg => arg !== '--skip-poll');
```

Comment:
Please use the arg parser from node:util

We have been burned many times by janky arg parsers